imod/protocols/protocol_importSetOfTM.py

```diff
@@ -187,8 +190,8 @@ def _validate(self):
         if matchingFiles:
             tsIdList = self.getInputSet().getTSIds()
             tmFileList = [normalizeTSId(fn) for fn, _ in self.iterFiles()]
-            self.matchingFiles = list(set(tsIdList) & set(tmFileList))
-            if not self.matchingFiles:
+            self.matchingTsIds = list(set(tsIdList) & set(tmFileList))
+            if not self.matchingTsIds:
```

Comment:
I assume we don't want to throw an error anymore? Then self.matchingTsIds can be defined in the self._initialize()

Reply:
In the end, in that part I only renamed the matchingFiles class attribute by matchingTsIds for clarity. But I can undo that part. It does not affect the name matching.

Reply:
Ok, I need to read the whole code properly. I'll do it later tonight.

Reply:
No worries. Many thanks, Grigory.